app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= simple_form_for @group do |f| %>
+  <% if @group.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@group.errors.count, "error") %> prohibited this group from being saved:</h2>
+
+      <ul>
+        <% @group.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= f.input :name %>
+    <%= f.input :description, as: :text %>
+    <%= f.input :amount %>
+    <%= f.association :category %>
+    <div id="addParticipants">
+    <%= link_to_add_association f, :participating_users do  %>
+      Agregar participante
+    <% end %>
+  </div>
+
+  <div class="participants">
+    <%= f.simple_fields_for :participating_users do |g| %>
+      <%= render  'participating_user_fields', f:g %>
```

Comment:
I observe that in your view the users do not appear in the select, do you plan to fix this in another issue?

![imagen](https://user-images.githubusercontent.com/12243331/143364453-92e8e864-5342-4515-b916-e3bede198e91.png)

Reply:
yes yil, in this project we have the plan to will work step by step and this observation are will resolve in the future